src/osdc/Objecter.h

```diff
@@ -1154,7 +1154,7 @@ struct ObjectOperation {
     for (uint32_t i = 0; i < sops.size(); i++) {
       out_bl[i] = &sops[i].outdata;
       out_handler[i] = NULL;
-      out_rval[i] = &sops[i].rval;
+      out_rval[i] = (int*)(&sops[i].rval);
```

Comment:
@liewegas 
Clang complains about conversion when there is no explicit cast to (int32*).
So either we add a a function that helps Clang, or we just cast?